va/drm/va_drm_utils.c

```diff
@@ -41,8 +41,8 @@ static const struct driver_name_map g_driver_name_map[] = {
     { "pvrsrvkm",   8, "pvr"    }, // Intel UMG PVR driver
     { "emgd",       4, "emgd"   }, // Intel ECG PVR driver
     { "hybrid",     6, "hybrid" }, // Intel OTC Hybrid driver
-    { "nouveau",    7, "gallium" }, // Mesa Gallium driver
-    { "radeon",     6, "gallium" }, // Mesa Gallium driver
+    { "nouveau",    7, "nouveau"  }, // Mesa Gallium driver
+    { "radeon",     6, "r600"     }, // Mesa Gallium driver
```

Comment:
Julien's support for nouveau st/va landed in 11.x.  When 12 release rolled around, gallium_drv_video.so was replaced with hard links to the respective backends.  These patches from you keep VA consistent with the current Mesa release.  Prior to this point it was an inconsistent level of support.  So I'm okay with this change based on the Mesa 12 commits which imply a switch to the platform names.  lgtm.